### DISCUSSION TITLE
How to handle screensaver/idle

### DISCUSSION THREAD

**Discussion by CZonin** [awawa-dev/HyperHDR#905] [Q&A]
I have HyperHDR running on Windows 11 with a HyperSerialESP32 controller. I'm using a QD-OLED monitor and to avoid burn-in, I use a blank screensaver that activates after 5 minutes of inactivity. About half the time when I get back to my PC, and end the screensaver, HyperHDR doesn't turn back on and I need to restart it. The only thing I see in logs is `The device is temporarily unavailable. Will try in 5 seconds again.`  Any ideas on how I can better handle screensavers/idle with HyperHDR?

**Comment by awawa-dev** [comment]
Need full logs (uploaded on pastebin on something similar) that include the moment when the screensaver kicks-in and it's removed: give it some time 1-2 minute then and try to update the desktop with anything eg. moving a windows to force changes and restart of capturing.

**Comment by CZonin** [comment]
Here's logs from when HyperHDR worked after returning from a screensaver: https://pastebin.com/mSK1yMxK
I'll keep monitoring it and try to get logs after it stops working.

**Comment by CZonin** [comment]
Finally got some logs for when HyperHDR didn't work after a screensaver: https://pastebin.com/SjT08cwQ

**Comment by awawa-dev** [comment]
>2024-07-17T12:48:40.217Z [MAIN] Starting HyperHdr - 20.0.0.0, (HEAD detached at v20.0.0.0) (awawa-dev-9e960c9/52f12b4-1711236142), built: Mar 23 2024:23:29:04
...
2024-07-17T12:48:40.300Z [HYPER_MANAGER] (HyperHdrManager.cpp:181) OS event: Monitor On
2024-07-17T12:48:40.300Z [HYPER_MANAGER] The system is going to wake up

The HyperHDR was started in the same moment when it received 'wake up' command. Sorry, that does not make any sense for me.

**Reply by CZonin** [reply]
Any ideas on what else I could do to find out what's causing the issue?

**Reply by awawa-dev** [reply]
No, despite this strange event at the beginning, the capturing was working. Then suddenly stopped without any notification from the system (there was no sleep signal or monitor disconnecting)

> 2024-07-17T12:53:40.199Z [PERFORMANCE] [INSTANCE0: FPS = 125.00, processed = 7500], [LED0: FPS = 58.82, send = 3529, processed = 3529, dropped = 0]
2024-07-17T12:54:37.030Z [MUXER0] Priority 245 is now inactive
